packages/nestjs/src/index.ts

```
@@ -0,0 +1,318 @@
+  constructor(
+    public name: string,
+    private meta: Record<string, any>,
+    private logger: NestLogger,
+    private isDebug: boolean,
+  ) {}
+  private prepareMessage(args: LazyLoggerMessage[]) {
+    const obj = {
+      ...(this.meta || {}),
+    };
+    const strs: string[] = [];
+    args = args
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)